src/app/domain/market/MavenProductInfo.php

```diff
@@ -38,6 +38,7 @@ public function getMavenArtifacts(string $version): array
                   ->type($project->type ?? 'iar')
                   ->makesSenseAsMavenDependency(false)
                   ->doc(false)
+                  ->archivedArtifact($project->archivedArtifact ?? [])
```

Comment:
it seems that you can have multiple archived artifacts, then we use in this code base plurarl:

```
archivedArtifacts($project->archivedArtifacts ?? [])